eventsource/request-cache-control.any.js

```diff
@@ -4,22 +4,32 @@
                     .replace(/\/[^\/]*$/, '/')
 
       // running it twice to check whether it stays consistent
-      function cacheTest(runAgain, url) {
-        var test = async_test()
+      function cacheTest(url) {
+        var test = async_test(url + "1")
+        // Recursive test. This avoids test that timeout
```

Comment:
This file seems to get stuck on a timeout.
@andreubotella can probably explain this change better than I can.

Reply:
Each of the top-level calls to `cacheTest` created two async tests, one at the moment of calling the function with `runAgain` set to `true`, and once again when that test succeeded. However, since the second test for each URL was created asynchronously, they might be created after the `load` event fires, and so the test might have completed by that time. This seemed to mostly work on the test as it was, but changing the script from an inline script to a fetch seemed to break this.

This change instead creates all async tests before the `load` event.